CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ The present file will list all changes made to the project; according to the
 - `DBMysql::affected_rows()`
 - `DBMysql::free_result()`
 - `DBMysql::isMySQLStrictMode()`
+- `getAllDatasFromTable` renamed to `getAllDataFromTable()`
+- Drop support of `$order` parameter in `getAllDataFromTable()` (`DbUtils::getAllDataFromTable()`)
```

Comment:
Reading this, I thought it was already dropped. I suggest this change to have a less ambiguous sentence.
```suggestion
- Usage of `$order` parameter in `getAllDataFromTable()` (`DbUtils::getAllDataFromTable()`)
```